planetiler-core/src/main/java/com/onthegomap/planetiler/ForwardingProfile.java

```diff
@@ -81,13 +81,21 @@ protected ForwardingProfile(Handler... handlers) {
   }
 
   private boolean caresAboutLayer(String layer) {
-    return (onlyLayers.isEmpty() || onlyLayers.contains(layer)) && !excludeLayers.contains(layer);
+    return ((onlyLayers.isEmpty() || onlyLayers.contains(layer)) && !excludeLayers.contains(layer)) ||
+      this.handlers.stream()
+        .filter(h -> h instanceof HandlerForLayer)
+        .map(h -> (HandlerForLayer) h)
```

Comment:
minor: filter-out this layer from the handlers?

Reply:
Adjusted.